### DISCUSSION TITLE
SV Max temp setting - I need help

### DISCUSSION THREAD

**Discussion by Derrillw** [artisan-roaster-scope/artisan#1417] [General]
I have a Kaliedo M2 Roaster and controlling it with Artisan Scope.    I have it configured and it seems to run the machine fine but I cannot roast any coffee because the SV slider max temp is set to 250def F.    I spent an hour yesterday trying to find the configuration setting in Artisan to raise this to 450 deg F.   Someone please tell me where this config setting is hidden.    I have searched and combed through the Artisan documentation and I cannot fine anything.

**Comment by Terracotta-6** [comment]
![ebde637eb4196219501dcd23af1e89f](https://github.com/artisan-roaster-scope/artisan/assets/149906183/f780ecee-3a5d-4e22-bb0e-756f7cfcf65c)

**Reply by Derrillw** [reply]
OK, much appreciated.
